lighthouse-core/test/audits/metrics-test.js

```diff
@@ -42,7 +42,7 @@ describe('Performance: metrics', () => {
       },
     };
 
-    const context = {settings: {throttlingMethod: 'devtools'}, computedCache: new Map()};
+    const context = {settings: {throttlingMethod: 'simulate'}, computedCache: new Map()};
```

Comment:
i don't know the context here but this stands out.... is this deliberate?

Reply:
it was devtools before because simulating wasn't an option for LCP :)

the other tests here use simulate

Reply:
gotcha ok